ocs_ci/deployment/deployment.py

```diff
@@ -561,11 +563,23 @@ def deploy_ocs_via_operator(self, image=None):
             templating.dump_data_to_temp_yaml(multus_data, multus_data_yaml.name)
             run_cmd(f"oc create -f {multus_data_yaml.name}")
 
+        enable_addon = config.DEPLOYMENT.get("live_deployment_enable_addon")
+
         if config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM:
             ibmcloud.add_deployment_dependencies()
             if not live_deployment:
                 create_ocs_secret(self.namespace)
-        self.subscribe_ocs()
+        if (
+            config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM
+            and live_deployment
+            and enable_addon
```

Comment:
I would maybe rather go with the opposite approach  and have disable_addon config value and configuration file for actually disable this addon for live deployment if we need it. So by default it will be by addon when live deployment.

WDYT?

Reply:
@petr-balogh : you mean to say if live_deployment, deploy with add-on by default?